rosetta-maven-plugin/pom.xml

```diff
@@ -59,6 +59,16 @@
 			<groupId>org.eclipse.xtext</groupId>
 			<artifactId>xtext-maven-plugin</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.regnosys.rosetta</groupId>
+			<artifactId>com.regnosys.rosetta.ide</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.regnosys.rosetta</groupId>
+			<artifactId>com.regnosys.rosetta.lib</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
This one should actually already be included transitively by `<artifactId>com.regnosys.rosetta</artifactId>`. Are you sure you need it?